.github/workflows/go.yml

```diff
@@ -38,20 +32,16 @@ jobs:
       run: go test ./... -race -coverprofile=coverage.txt -covermode=atomic
 
     - name: Coverage
-      uses: codecov/codecov-action@125fc84a9a348dbcf27191600683ec096ec9021c # v4.4.1
+      uses: codecov/codecov-action@125fc84a9a348dbcf27191600683ec096ec9021c # v4.3.1
```

Comment:
Was this downgrade intentional?  Looks like the new hash is mapping to an older release.

Reply:
Ah, no, I messed up the rebase on accident. Thanks for the catch.